jest.config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is this file necessary? if so, why?

Reply:
Not necessary. I just removed it. 

Context: It was there to help with the debugging tools, but it wasn't loading the proper configuration for each directory. Solved [here](https://github.com/decentraland/catalyst/pull/847/commits/20fb1762cf9267ee80d5a0e3869c428eecd998e6)